partials/content.php

```diff
@@ -35,7 +35,7 @@
 		<?php the_excerpt(); ?>
 	</div>
 	<?php else : ?>
-	<div class="entry-content" itemprop="articleBody">
+	<div class="entry__content entry__a--transition" itemprop="articleBody">
```

Comment:
Ok makes more sense, so we are going to remove all ` entry__a--transition.` right?